### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by leandroyalet**
Refs #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,24 +29,48 @@
...
+// 4. Create peer information
+var peerInfo = await RifCommunications.createPeerInfo(peerId);
+
+// 5. Create your peer node
+var node = RifCommunications.createNode(peerInfo, host, port, sendMsgFunc);
+
+// 6. Connect to a bootnode using a multiaddress string
+await RifCommunications.connectToNode(node, BOOTNODE_ADDRESS);
+
+// 7. From the public key of the recipient create the recipient
```
Comment (jurajpiar) [inline README.md]: Perhaps clearer:
> Create recipient from their public key (`<recipientID>`)

File: README.md [hunk 2]
```diff
@@ -30,24 +29,48 @@
...
+var peerId = await RifCommunications.createPeerIdFromJSON(key);
+
+// 4. Create peer information
+var peerInfo = await RifCommunications.createPeerInfo(peerId);
+
+// 5. Create your peer node
+var node = RifCommunications.createNode(peerInfo, host, port, sendMsgFunc);
+
+// 6. Connect to a bootnode using a multiaddress string
+await RifCommunications.connectToNode(node, BOOTNODE_ADDRESS);
```
Comment (jurajpiar) [inline README.md]: Maybe would be good to make clear that this is the initialisation part with sub heading, e.g:
## Usage
### Initialise
`...`
### Connect to another user
`...`

File: README.md [hunk 3]
```diff
@@ -30,24 +29,48 @@
...
+
 ## API
 
 ## Contribute
 
 There are some ways you can make this module better:
 
-- Consult our [open issues](https://github.com/rsksmart/rif-communications-js/issues) and take on one of them
+- Consult our [open issues](https://github.com/rsksmart/rif-communications/issues) and take on one of them
 - Help our tests reach 100% coverage!
```
Comment (jurajpiar) [inline README.md]: > How you can improve this module:
> - Consult our [open issues](https://github.com/rsksmart/rif-communications/issues) page and pick one
> - Help us reach 100% test coverage!

Comment (vojtechsimetka) [reply README.md]: Isn't it still `rif-communications-js/issues`? Why was this line changed?

Comment (leandroyalet) [reply README.md]: I think same @jurajpiar but that is a part of our Readme template, we can discuss about this. @vojtechsimetka: I don't know how it happened

**Pull request merged**